src/helpers/validator.ts

```diff
@@ -75,7 +75,7 @@ export const isMessageType = (
 ): messageType is MessageTypes => [1, 2, 3].includes(messageType);
 
 export const validateMessage = (
-  message: string,
+  message: unknown,
```

Comment:
I think it's ok to keep the argument as `string`. Just add `ts-ignore rule` in the test file to get rid of warnings.

Reply:
Or maybe even use a separate tsconfig/eslint configuration that doesn't force specific rules. These rules make sense for the project source code but not always for unit tests.